prac_06/guitars.py

```diff
@@ -0,0 +1,24 @@
+"""
+CP5632 Practical 6
+Estimate: 30 minutes
+Actual:   36 minutes
+"""
+
+from guitar import Guitar
+
+guitars = []
+print("My guitars!")
+name = input("Name: ")
+while name != "":
```

Comment:
tip : you can imlement the getting of name here and save 2 lines of code :
_while (name := input("Name: ")) != "":_

Reply:
Thank you very much for your help！